src/twisted/internet/defer.py

```diff
@@ -28,11 +29,13 @@
 
 # Twisted imports
 from twisted.python.compat import cmp, comparable
-from twisted.python import lockfile, failure
+from twisted.python import lockfile, failure, reflect
 from twisted.logger import Logger
 from twisted.python.deprecate import warnAboutFunction, deprecated
 
 log = Logger()
+_contextvars = reflect.requireModule('contextvars')
```

Comment:
Oof. Given our aspirations to split out `deferred`, `twisted.python.reflect` seems like potentially quite a heavy dependency to take on.

How do you feel about this, instead:

```suggestion
try:
    from contextvars import copy_context as _copy_context
except ImportError:
    class _NoContext:
        @staticmethod
        def run(f, *args, **kwargs):
            return f(*args, **kwargs)
    def _copy_context():
        return _NoContext
```